.github/workflows/prepare-0.13.2.yml

```
@@ -0,0 +1,73 @@
+          path = Path('CHANGELOG.md')
+          text = path.read_text()
+          anchor = '## [0.13.1] - 2026-08-16\n'
+          if text.count(anchor) != 1:
+              raise SystemExit('0.13.1 changelog anchor is not unique')
+          if '## [0.13.2] - 2026-08-16' in text:
+              raise SystemExit('0.13.2 section already exists')
+
+          section = '''## [0.13.2] - 2026-08-16
+
+### Fixed
+
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)